CHANGES.md

```diff
@@ -1,6 +1,7 @@
 # UNRELEASED
 
 - [FIXED] Set default value for `partitioned` parameter to false when creating a design document.
+- [FIXED] Always set `partitioned` for `create_query_index` requests
```

Comment:
I think this could be a little confusing for a user because it could imply that we're always setting `partitioned=True` and nit for a trailing period/full stop.
How about:
```
- [FIXED] -  Corrected setting of `partitioned` flag for `create_query_index` requests.
```

Reply:
Done in 675de0d .